ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -168,8 +168,7 @@ public static boolean checkDistributionAvailable(String distribution) {
         conn.setRequestProperty("Accept", "application/json");
         if (conn.getResponseCode() != 200) {
             conn.disconnect();
-            throw ErrorUtil.createCommandException("server request failed. HTTP error code " +
-                                                           conn.getResponseCode());
+            throw ErrorUtil.createCommandException("server request failed: " + conn.getResponseMessage());
```

Comment:
Can the getResponseMessage be null in any case?

Reply:
Updated with d28f662 to use status code just in case the message is null.